coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -336,6 +377,57 @@ export default {
 
     isCompleted (item) {
       return item.status === 'COMPLETED'
+    },
+
+    confirmDeleteDraft (item) {
+      // open confirmation dialog and wait for response
+      this.$refs.confirm.open(
+        'Delete Draft?',
+        'Delete your ' + item.draftTitle + '? Any changes you\'ve made will be lost.',
+        { width: '40rem',
+          persistent: true,
+          yes: 'Delete',
+          no: null,
+          cancel: 'Don\'t delete',
+          className: 'delete-draft-confirmation-dialog' }
+      ).then(async (confirm) => {
+        // if we get here, Yes or No was clicked
+        if (confirm) {
+          await this.doDeleteDraft(item)
+        } else {
+          // do nothing
+        }
+      }).catch(() => {
+        // if we get here, Cancel was clicked - do nothing
+      })
+    },
+
+    async doDeleteDraft (item) {
+      let url = this.entityIncNo + '/filings/' + item.id
+      await axios.delete(url).then(res => {
+        if (!res) { throw new Error('invalid API response') }
+
+        // reload dashboard
+        this.setTriggerDashboardReload(true)
```

Comment:
In other places, the code simply routes to the Home URL, which reloads everything.
```
this.$router.push('/')
```
Do you think that could work here instead of setting a variable that is watched?

Reply:
It didn't work when I was testing without the actual api, but actually let me check again now that I can run the api locally with that delete action implemented...

Reply:
Nope just tested the router push, didn't work - no change on screen evidenced at all.